src/Protocol/ActivityPub/Delivery.php

```diff
@@ -84,7 +84,7 @@ public static function deliverToInbox(string $cmd, int $item_id, string $inbox,
 			if (empty($item['id'])) {
 				Logger::warning('Item not found, removing delivery', ['uri-id' => $uri_id, 'uid' => $uid, 'cmd' => $cmd, 'inbox' => $inbox]);
 				Post\Delivery::remove($uri_id, $inbox);
-				return true;
+				return ['success' => true, 'serverfailure' => false, 'drop' => false];
```

Comment:
Okay, nice find. The returned type `bool` would fail with `array` as set returned type. But:
1) Is "successful" really true here, it feels more like failed to me.
2) `empty($item['id'])` can be replaced with `!DBA::isResult($item)`?

Reply:
When the item doesn't exist anymore we mustn't retry. We retry on failure. So we define this a success.